sparta/sparta/ports/SyncPort.hpp

```diff
@@ -158,7 +158,7 @@ namespace sparta
 
             OutPort::bind(in);
 
-            bound_ports_.push_back(in);
+            bound_in_ports_.push_back(inp);
```

Comment:
Ugh, good catch -- old code was storing the in port in the same vector -- twice!  I know you needed a new vector...